azurerm/internal/services/servicefabric/resource_arm_service_fabric_cluster.go

```diff
@@ -273,6 +273,123 @@ func resourceArmServiceFabricCluster() *schema.Resource {
 				},
 			},
 
+			"upgrade_description": {
+				Type:     schema.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"force_restart": {
+							Type:     schema.TypeBool,
+							Optional: true,
+						},
+						"health_check_retry_timeout": {
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+						"health_check_stable_duration": {
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+						"health_check_wait_duration": {
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+						"upgrade_domain_timeout": {
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+						"upgrade_replica_set_check_timeout": {
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+						"upgrade_timeout": {
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+						"health_policy": {
+							Type:     schema.TypeList,
+							Optional: true,
+							MaxItems: 1,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"max_percent_unhealthy_applications": {
```

Comment:
coulw we move percent to be at the end?
```suggestion
									"max_unhealthy_applications_percent": {
```